hr_worked_days_hourly_rate/README.rst

```diff
@@ -0,0 +1,51 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+Worked Days Hourly Rate
+=====================================================
```

Comment:
The line should be of the same length of the title, and put it also in the upper part of the title